pytest_operator/plugin.py

```diff
@@ -1101,27 +1102,51 @@ def abort(self, *args, **kwargs):
         self.aborted = True
         pytest.fail(*args, **kwargs)
 
+    @overload
+    async def build_charm(
+        self,
+        charm_path,
+        bases_index: Optional[int] = None,
+        verbosity: Optional[
+            Literal["quiet", "brief", "verbose", "debug", "trace"]
+        ] = None,
+        return_all: Literal[False] = False,
+    ) -> Path: ...
+
+    @overload
+    async def build_charm(
+        self,
+        charm_path,
+        bases_index: Optional[int] = None,
+        verbosity: Optional[
+            Literal["quiet", "brief", "verbose", "debug", "trace"]
+        ] = None,
+        return_all: Literal[True] = True,
+    ) -> List[Path]: ...
+
```

Comment:
These overload implementations are here for type hinting. 
If you pass a `return_all=True` you get a `List[Path]`
If you pass a `return_all=False` you get a `Path`

Reply:
> [If there are multiple equally good matching variants, mypy will select the variant that was defined first.](https://mypy.readthedocs.io/en/stable/more_types.html#type-checking-calls-to-overloads)

so, ```ops_test.build_charm(path)``` will yield the same `Path` return type.